### ISSUE TITLE
Failed to handle memberRemoved: NullPointerException

### ISSUE THREAD

**Issue by anhptvolga** [vert-x3/vertx-hazelcast#162]
### Version

vertx 4.3.4

### Context

I encountered an exception from our sentry.
We use vertx cluster in kubernetes.

```
java.lang.NullPointerException: Cannot invoke "io.vertx.spi.cluster.hazelcast.impl.SubsMapHelper.removeAllForNodes(java.util.Set)" because "this.subsMapHelper" is null
    at io.vertx.spi.cluster.hazelcast.HazelcastClusterManager.cleanSubs(HazelcastClusterManager.java:319)
    at io.vertx.spi.cluster.hazelcast.HazelcastClusterManager.membersRemoved(HazelcastClusterManager.java:307)
    at io.vertx.spi.cluster.hazelcast.HazelcastClusterManager.memberRemoved(HazelcastClusterManager.java:300)
    at com.hazelcast.internal.cluster.impl.ClusterServiceImpl.dispatchEvent(ClusterServiceImpl.java:758)
    at com.hazelcast.internal.cluster.impl.ClusterServiceImpl.dispatchEvent(ClusterServiceImpl.java:91)
    at com.hazelcast.spi.impl.eventservice.impl.LocalEventDispatcher.run(LocalEventDispatcher.java:64)
    at com.hazelcast.internal.util.executor.StripedExecutor$Worker.process(StripedExecutor.java:245)
    at com.hazelcast.internal.util.executor.StripedExecutor$Worker.run(StripedExecutor.java:228)
```

### Extra

jdk 17

**Comment by tsegismont** [comment]
Thanks reporting this, it will be fixed in the next bug fix release

**Comment by tsegismont** [comment]
Fixed in 03da861
